src/pages/plugins/plugins.module.scss

```diff
@@ -31,6 +30,63 @@
   font-size: 1.1rem;
 }
 
+.pluginDeprecatedCard {
+  border-radius: var(--ifm-card-border-radius);
+  box-shadow: var(--ifm-global-shadow-lw);
+  overflow: hidden;
+  background-color: #ffdfdf;
+
+  .pluginCardHeader {
```

Comment:
Don't nest these styles. We use CSS Modules so we don't need to worry about nesting.